lib/std/zig/render.zig

```diff
@@ -384,22 +384,9 @@ fn renderExpression(r: *Render, node: Ast.Node.Index, space: Space) Error!void {
 
         .block_two,
         .block_two_semicolon,
-        => {
-            const statements = [2]Ast.Node.Index{ datas[node].lhs, datas[node].rhs };
-            if (datas[node].lhs == 0) {
-                return renderBlock(r, node, statements[0..0], space);
-            } else if (datas[node].rhs == 0) {
-                return renderBlock(r, node, statements[0..1], space);
-            } else {
-                return renderBlock(r, node, statements[0..2], space);
-            }
-        },
         .block,
         .block_semicolon,
-        => {
-            const statements = tree.extra_data[datas[node].lhs..datas[node].rhs];
-            return renderBlock(r, node, statements, space);
-        },
```

Comment:
I don't like this change. I did it that way on purpose, so that there would be exactly one branch on the ast node tag. Now there are two. I don't find addition of `renderAnyBlock` to be any more readable either. Previously all the logic for abstracting over the different encodings was grouped in `renderExpression`; now it is spread out more.

Reply:
I understand.

The reason for `renderAnyBlock` is the new `discards` prameter, which currently is only used for unbounded for-range captures. I could probably bind the discard fixup to the block node in advance, then the parameter won't be needed. Sounds good?

Reply:
> One solution I can think of is to briefly store the `discards` slice on the fixups struct. `renderBlock` would make use of it and reset it to `null`.

I went ahead and did this, minus the nullability.

Reply:
moved from fixups to render